commit/plugin.go

```diff
@@ -79,6 +80,18 @@ func NewPlugin(
 		offchainCfg.MaxMerkleTreeSize = merklemulti.MaxNumberTreeLeaves
 	}
 
+	if offchainCfg.RMNEnabled {
+		if err := rmnHomeReader.Ready(); err == nil {
```

Comment:
What does `Ready` do exactly?

Reply:
Checks that the service has not been started yet, which in this case should not be happening.